test/fixtures/valid.js

```diff
@@ -1,4 +1,5 @@
 // Rule: linebreak-style
+// Rule: max-len
```

Comment:
This is not a real test :)

Let's add a regular line of code that (with indentation) just about exceeds 100 chars. And ensure it passes with eslint-disable-next-line over it, and also add the expected failure to the invalid fixture.

We should also have a second test that confirms really long `QUnit.test()` lines are allowed without explicit disable rule, and that long `// eslint-` lines are allowed as well.